masterlist.yaml

```diff
@@ -10644,6 +10661,7 @@ plugins:
       - 'BecomeKingofSkyrim.esp'
       - 'BetterQuestObjectives.esp'
       - 'Complete Alchemy & Cooking Overhaul.esp'
+      - 'Falskaar.esm'
```

Comment:
We shouldn't really need that after as Falskaar is an ESM flagged plugin and will sort before ASLAL by default.

```suggestion
```